indexer-core/pom.xml

```diff
@@ -84,21 +84,25 @@ under the License.
     <dependency>
       <groupId>org.apache.lucene</groupId>
       <artifactId>lucene-core</artifactId>
+      <version>6.1.0</version>
```

Comment:
Hi,

Just a small remark, if I may -- this is not the right place to be changing the version. There is a version property defined in the aggregator's `pom.xml`. You should be changing that instead.

Cheers,

Martin